service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/ServiceLocator.scala

```diff
@@ -3,11 +3,9 @@
  */
 package com.lightbend.lagom.scaladsl.api
 
-import java.net.{ URI, URISyntaxException }
+import java.net.URI
 
 import com.lightbend.lagom.scaladsl.api.Descriptor.Call
-import com.typesafe.config.ConfigException
-import play.api.Configuration
 
 import scala.concurrent.{ ExecutionContext, Future }
```

Comment:
I stepped over several parts of the code with some obsolete imports and took a chance to do a cleanup. (there's more file changes like this one)